UIforETW/UIforETWDlg.cpp

```diff
@@ -769,7 +774,7 @@ void CUIforETWDlg::OnBnClickedStarttracing()
 	else
 		UIETWASSERT(0);
 
-	std::wstring kernelProviders = L" Latency+POWER+DISPATCHER+DISK_IO_INIT+FILE_IO+FILE_IO_INIT+VIRT_ALLOC+MEMINFO";
+	std::wstring kernelProviders = L" Base+Latency+POWER+DISPATCHER+DISK_IO_INIT+FILE_IO+FILE_IO_INIT+VIRT_ALLOC+MEMINFO";

```

Comment:
Added it because it was in the example:
https://msdn.microsoft.com/en-us/library/windows/hardware/hh448186.aspx
I still get symbols without it so it seems to be unnecessary.